egs2/librimix/enh_diar1/local/data.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+stage=0       # start from 0 if you need to start from data preparation
+stop_stage=100
+FOLDER=git_librimix
+fs=8k
+num_spk="2 3"
+min_max_mode=min
+adapt=
+
+ . utils/parse_options.sh || exit 1;
+
+
+if [ -z "${LIBRIMIX}" ]; then
+    log "Fill the value of 'LIBRIMIX' of db.sh"
+    exit 1
+fi
+mkdir -p ${LIBRIMIX}
+
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+log "data preparation started"
+
+# Github LibriMix : https://github.com/s3prl/LibriMix.git
+if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ] ; then  
```

Comment:
I prefer to keep the current code since they are slightly different. `enh_diar` has extra files for the enhancement task (`spk{1,2,3}.scp` and `noise1.scp`)